apps/api/src/app/integrations/usecases/check-integration/check-integration.command.ts

```diff
@@ -6,8 +6,10 @@ export class CheckIntegrationCommand extends EnvironmentCommand {
   @IsDefined()
   @IsString()
   providerId: string;
+
   @IsDefined()
   channel: ChannelTypeEnum;
+
   @IsDefined()
-  credentials: ICredentials;
+  credentials?: ICredentials;
```

Comment:
Another idea. Maybe we shouldn't allow in the UI to check integration if the credentials are not set for the integration.